.github/config/evergreen-arm-hardfp-raspi.json

```diff
@@ -7,7 +7,9 @@
     "test_root_target": "//cobalt:gn_all",
     "num_gtest_shards": 1,
     "targets": [
-      "loader_app"
+      "cobalt:gn_all",
```

Comment:
Can you explicitly add "cobalt_loader" to the `cobalt:gn_all` target for hermetic builds? I don't think it'll build with just this